### ISSUE TITLE
Music Images Not Loading

### ISSUE THREAD

**Issue by NeoShader** [hymns-for-worship/issues#68]
Sheet music not loading. Possible S3 issue?

Exception: Invalid image data

![Screenshot_2023-08-07-21-18-43-44_0a652b863657546cdae25450fc60e6b8](https://github.com/hymns-for-worship/issues/assets/18433005/23c52843-e030-49d4-a8c6-c43d9292b03e)

**Comment by Graywolf6953** [comment]
I am having this issue also. Working one day and next song will not load

**Comment by brdillman** [comment]
This is happening on my tablet and phone. The occasional song will show the sheet music.

**Comment by Graywolf6953** [comment]
Same here. Getting frustrating.

**Comment by CatherineHebert** [comment]
Same.  More than frustrated.  I find that every time someone does an "update" my app no longer works for me.  I've never had another app that makes your version stop working because an update was done.  And now even after updating it still won't work.  After updating it won't even let me type in the first number of a song - it jumps me out.

**Comment by Graywolf6953** [comment]
Same issue here. I am waiting for this next update but it is not coming.
This is the only app I have that stops working every time there is an
update except this time I had not done an update and it just
stopped working.

**Comment by rodydavis** [comment]
2.2.10 and 2.3.0 just rolled out and fixes this!

TLDR, there was a server update and google play publishing delay on rollout. There is not a mechanism for android to preform critical in app updates in the future.

**Comment by rodydavis** [comment]
I really apologize for the delay and broken update experience. This last update was critical for the Linux and windows desktop apps I will be launching and the current database and auth libraries were not working.

the new auth flow should be much more flexible and a nice experience. Even trying to get pass keys working too.

thank you for the feedback and I hope to address current and future concerns

**Comment by Graywolf6953** [comment]
It appears my account, I had since 2022, has somehow disappeared. I
requested to change my password. With my email address it sends me a link
to reset. I do that and log in only to find my profile is gone and a
strange user name in its place. (users58329) does anyone recognize that
user name?

**Comment by rodydavis** [comment]
> It appears my account, I had since 2022, has somehow disappeared. I
> 
> requested to change my password. With my email address it sends me a link
> 
> to reset. I do that and log in only to find my profile is gone and a
> 
> strange user name in its place. (users58329) does anyone recognize that
> 
> user name?

You need to create an account with the same email to have it transfer!

**Comment by Graywolf6953** [comment]
The email I am using is the in one I was logged into before program stopped
working. I had NOT even done an update when it stopped this time. How do I
create an account using the same email address? I will not pay for the
program again.

**Comment by NeoShader** [comment]
Same here. I no longer have paid HFW, HFWS, or echoice. It shows me as
basic.

**Comment by Graywolf6953** [comment]
That is all mine shows too. I am definitely not an expert but I suspect the
problem is with the server. I don't know. Just would like to see it fixed.

**Comment by Graywolf6953** [comment]
Can we just get a hymnal that will pull up selected song and act like a
song book? My bible program works that way and I am happy without all the
bells and whistles.

**Comment by rodydavis** [comment]
Creating an account is free, you just sign up with the same email.

I am also looking into a bug with some purchases not restoring for some users.

Like I said earlier this was in preparation for the windows app that will be released soon. The existing solution was used for many years since launch.

**Comment by rodydavis** [comment]
I am working on a simpler interface. But a big reason for a lot of the changes too is due to the licensing and splitting the viewing sheet music / slides and exporting power point into different plans

**Comment by Graywolf6953** [comment]
How long does it take to create the account? I did what was requested withy
email and it has been working on it for 15 minutes now.

**Comment by rodydavis** [comment]
> Same here. I no longer have paid HFW, HFWS, or echoice. It shows me as
> 
> basic.

Hymnals and eChoice are merged into basic plan now.
